2020-04-11 17:34:55,643 : INFO : Machine : Mission Pinball Framework Core Engine v0.52.9
2020-04-11 17:34:55,643 : INFO : Machine : Command line arguments: {'create_config_cache': True, 'configfile': ['config.yaml'], 'jsonlogging': False, 'production': False, 'pause': False, 'text_ui': True, 'force_assets_load': False, 'bcp': True, 'mc_file_name': None, 'consoleloglevel': 20, 'loglevel': 15, 'no_load_cache': False, 'syslog_address': None, 'no_sound': False, 'logfile': 'logs\\2020-04-11-17-34-55-mpf-DESKTOP-I5I6D25.log', 'force_platform': None, 'mpfconfigfile': 'c:\\users\\mission pinball\\appdata\\local\\programs\\python\\python35\\lib\\site-packages\\mpf\\mpfconfig.yaml'}
2020-04-11 17:34:55,658 : INFO : Machine : MPF path: c:\users\mission pinball\appdata\local\programs\python\python35\lib\site-packages\mpf
2020-04-11 17:34:55,658 : INFO : Machine : Machine path: E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS
2020-04-11 17:34:55,658 : INFO : Machine : Platform: win32
2020-04-11 17:34:55,658 : INFO : Machine : Python executable location: c:\users\mission pinball\appdata\local\programs\python\python35\python.exe
2020-04-11 17:34:55,658 : INFO : Machine : Python version: 3.5.4 (64-bit)
2020-04-11 17:34:55,658 : INFO : Machine : Machine config file #1: config.yaml
2020-04-11 17:34:55,674 : WARNING : ConfigProcessor : Cache file not found: C:\Users\MISSIO~1\AppData\Local\Temp\1d59add9599c514c5f7c5be61c73be75.mpf_cache
2020-04-11 17:34:55,674 : INFO : ConfigProcessor : Loading config from file c:\users\mission pinball\appdata\local\programs\python\python35\lib\site-packages\mpf\mpfconfig.yaml.
2020-04-11 17:34:57,814 : INFO : ConfigProcessor : Loading config: c:\users\mission pinball\appdata\local\programs\python\python35\lib\site-packages\mpf\mpfconfig.yaml
2020-04-11 17:34:58,064 : INFO : ConfigProcessor : Loading config from file E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\config\config.yaml.
2020-04-11 17:34:59,221 : INFO : ConfigProcessor : Loading config: E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\config\config.yaml
2020-04-11 17:34:59,393 : INFO : ConfigProcessor : Loading config: E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\config\player_variables.yaml
2020-04-11 17:34:59,518 : INFO : ConfigProcessor : Loading config: E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\config\key_mapping.yaml
2020-04-11 17:34:59,549 : INFO : ConfigProcessor : Config file cache created: C:\Users\MISSIO~1\AppData\Local\Temp\1d59add9599c514c5f7c5be61c73be75.mpf_cache
2020-04-11 17:34:59,564 : INFO : Machine : Initialise MPF.
2020-04-11 17:35:00,939 : INFO : FAST : Connecting to com4 at 921600bps
2020-04-11 17:35:01,002 : INFO : FAST : Connected! Processor: NET, Board Type: FP-CPU-002-2, Firmware: 01.01
2020-04-11 17:35:01,002 : INFO : EventManager : Event: ======'machine_var_fast_net_firmware'====== Args={'value': '01.01', 'prev_value': None, 'change': True}
2020-04-11 17:35:01,002 : INFO : EventManager : Event: ======'machine_var_fast_net_model'====== Args={'value': 'FP-CPU-002-2', 'prev_value': None, 'change': True}
2020-04-11 17:35:01,002 : WARNING : FAST : Not resetting FAST NET because firmware is currently broken.
2020-04-11 17:35:01,080 : INFO : FAST : Connecting to com5 at 921600bps
2020-04-11 17:35:01,127 : INFO : FAST : Connected! Processor: RGB, Board Type: FP-CPU-002-2, Firmware: 01.00
2020-04-11 17:35:01,127 : INFO : EventManager : Event: ======'machine_var_fast_rgb_firmware'====== Args={'value': '01.00', 'prev_value': None, 'change': True}
2020-04-11 17:35:01,143 : INFO : EventManager : Event: ======'machine_var_fast_rgb_model'====== Args={'value': 'FP-CPU-002-2', 'prev_value': None, 'change': True}
2020-04-11 17:35:01,143 : INFO : EventManager : Event: ======'machine_var_credits_string'====== Args={'value': 'FREE PLAY', 'prev_value': None, 'change': True}
2020-04-11 17:35:01,861 : INFO : EventManager : Event: ======'machine_var_player1_score'====== Args={'value': 107, 'prev_value': None, 'change': True}
2020-04-11 17:35:01,861 : INFO : EventManager : Event: ======'machine_var_mpf_version'====== Args={'value': 'MPF v0.52.9', 'prev_value': None, 'change': True}
2020-04-11 17:35:01,861 : INFO : EventManager : Event: ======'machine_var_mpf_extended_version'====== Args={'value': 'MPF v0.52.9, Config version:5, Show version: 5, BCP version:1.1', 'prev_value': None, 'change': True}
2020-04-11 17:35:01,861 : INFO : EventManager : Event: ======'machine_var_python_version'====== Args={'value': '3.5.4', 'prev_value': None, 'change': True}
2020-04-11 17:35:01,877 : INFO : EventManager : Event: ======'machine_var_platform'====== Args={'value': 'Windows-10-10.0.18362-SP0', 'prev_value': None, 'change': True}
2020-04-11 17:35:01,877 : INFO : EventManager : Event: ======'machine_var_platform_system'====== Args={'value': 'Windows', 'prev_value': None, 'change': True}
2020-04-11 17:35:01,877 : INFO : EventManager : Event: ======'machine_var_platform_release'====== Args={'value': '10', 'prev_value': None, 'change': True}
2020-04-11 17:35:01,877 : INFO : EventManager : Event: ======'machine_var_platform_version'====== Args={'value': '10.0.18362', 'prev_value': None, 'change': True}
2020-04-11 17:35:01,877 : INFO : EventManager : Event: ======'machine_var_platform_machine'====== Args={'value': 'AMD64', 'prev_value': None, 'change': True}
2020-04-11 17:35:01,877 : INFO : EventManager : Event: ======'init_phase_1'====== Args={}
2020-04-11 17:35:02,455 : WARNING : ConfigProcessor : Cache file not found: C:\Users\MISSIO~1\AppData\Local\Temp\e316602f2754df2a40ebb1b620c83c27.mpf_cache
2020-04-11 17:35:02,455 : INFO : ConfigProcessor : Loading config from file E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\gym3\config\gym3.yaml.
2020-04-11 17:35:02,643 : INFO : ConfigProcessor : Loading config: E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\gym3\config\gym3.yaml
2020-04-11 17:35:02,643 : INFO : ConfigProcessor : Config file cache created: C:\Users\MISSIO~1\AppData\Local\Temp\e316602f2754df2a40ebb1b620c83c27.mpf_cache
2020-04-11 17:35:02,814 : WARNING : ConfigProcessor : Cache file not found: C:\Users\MISSIO~1\AppData\Local\Temp\b9fac23b74e0b513c055e4621b74ad7e.mpf_cache
2020-04-11 17:35:02,814 : INFO : ConfigProcessor : Loading config from file E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\gym1_ended\config\gym1_ended.yaml.
2020-04-11 17:35:02,939 : INFO : ConfigProcessor : Loading config: E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\gym1_ended\config\gym1_ended.yaml
2020-04-11 17:35:02,955 : INFO : ConfigProcessor : Config file cache created: C:\Users\MISSIO~1\AppData\Local\Temp\b9fac23b74e0b513c055e4621b74ad7e.mpf_cache
2020-04-11 17:35:02,971 : WARNING : ConfigProcessor : Cache file not found: C:\Users\MISSIO~1\AppData\Local\Temp\4ea733655e2f71aef4d6f646b7999fe0.mpf_cache
2020-04-11 17:35:02,971 : INFO : ConfigProcessor : Loading config from file E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\gym5\config\gym5.yaml.
2020-04-11 17:35:03,189 : INFO : ConfigProcessor : Loading config: E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\gym5\config\gym5.yaml
2020-04-11 17:35:03,205 : INFO : ConfigProcessor : Config file cache created: C:\Users\MISSIO~1\AppData\Local\Temp\4ea733655e2f71aef4d6f646b7999fe0.mpf_cache
2020-04-11 17:35:03,236 : WARNING : ConfigProcessor : Cache file not found: C:\Users\MISSIO~1\AppData\Local\Temp\c81a0f1062502e33bbed66b9eae15858.mpf_cache
2020-04-11 17:35:03,252 : INFO : ConfigProcessor : Loading config from file E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\base\config\base.yaml.
2020-04-11 17:35:03,314 : INFO : ConfigProcessor : Loading config: E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\base\config\base.yaml
2020-04-11 17:35:03,314 : INFO : ConfigProcessor : Config file cache created: C:\Users\MISSIO~1\AppData\Local\Temp\c81a0f1062502e33bbed66b9eae15858.mpf_cache
2020-04-11 17:35:03,346 : WARNING : ConfigProcessor : Cache file not found: C:\Users\MISSIO~1\AppData\Local\Temp\ead1ab3e53925d446b3cb0becf429113.mpf_cache
2020-04-11 17:35:03,346 : INFO : ConfigProcessor : Loading config from file E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\pichu\config\pichu.yaml.
2020-04-11 17:35:03,736 : INFO : ConfigProcessor : Loading config: E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\pichu\config\pichu.yaml
2020-04-11 17:35:03,736 : INFO : ConfigProcessor : Config file cache created: C:\Users\MISSIO~1\AppData\Local\Temp\ead1ab3e53925d446b3cb0becf429113.mpf_cache
2020-04-11 17:35:03,768 : WARNING : ConfigProcessor : Cache file not found: C:\Users\MISSIO~1\AppData\Local\Temp\4c74177f109f95ed9599c78880646655.mpf_cache
2020-04-11 17:35:03,768 : INFO : ConfigProcessor : Loading config from file c:\users\mission pinball\appdata\local\programs\python\python35\lib\site-packages\mpf\modes\high_score\config\high_score.yaml.
2020-04-11 17:35:04,143 : INFO : ConfigProcessor : Loading config: c:\users\mission pinball\appdata\local\programs\python\python35\lib\site-packages\mpf\modes\high_score\config\high_score.yaml
2020-04-11 17:35:04,158 : INFO : ConfigProcessor : Loading config from file E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\high_score\config\high_score.yaml.
2020-04-11 17:35:04,893 : INFO : ConfigProcessor : Loading config: E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\high_score\config\high_score.yaml
2020-04-11 17:35:04,893 : INFO : ConfigProcessor : Config file cache created: C:\Users\MISSIO~1\AppData\Local\Temp\4c74177f109f95ed9599c78880646655.mpf_cache
2020-04-11 17:35:04,986 : WARNING : ConfigProcessor : Cache file not found: C:\Users\MISSIO~1\AppData\Local\Temp\5903aae11c08aba35176e5c1f4c1be59.mpf_cache
2020-04-11 17:35:04,986 : INFO : ConfigProcessor : Loading config from file E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\gym2_ended\config\gym2_ended.yaml.
2020-04-11 17:35:05,111 : INFO : ConfigProcessor : Loading config: E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\gym2_ended\config\gym2_ended.yaml
2020-04-11 17:35:05,111 : INFO : ConfigProcessor : Config file cache created: C:\Users\MISSIO~1\AppData\Local\Temp\5903aae11c08aba35176e5c1f4c1be59.mpf_cache
2020-04-11 17:35:05,174 : WARNING : ConfigProcessor : Cache file not found: C:\Users\MISSIO~1\AppData\Local\Temp\201769dfbc23cf51d44452c1bfe1f92e.mpf_cache
2020-04-11 17:35:05,174 : INFO : ConfigProcessor : Loading config from file E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\gym0\config\gym0.yaml.
2020-04-11 17:35:05,518 : INFO : ConfigProcessor : Loading config: E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\gym0\config\gym0.yaml
2020-04-11 17:35:05,518 : INFO : ConfigProcessor : Config file cache created: C:\Users\MISSIO~1\AppData\Local\Temp\201769dfbc23cf51d44452c1bfe1f92e.mpf_cache
2020-04-11 17:35:05,643 : WARNING : ConfigProcessor : Cache file not found: C:\Users\MISSIO~1\AppData\Local\Temp\8e537f3f1075ad5ff3f7e8ba8228812c.mpf_cache
2020-04-11 17:35:05,643 : INFO : ConfigProcessor : Loading config from file E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\gym3_ended\config\gym3_ended.yaml.
2020-04-11 17:35:05,768 : INFO : ConfigProcessor : Loading config: E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\gym3_ended\config\gym3_ended.yaml
2020-04-11 17:35:05,768 : INFO : ConfigProcessor : Config file cache created: C:\Users\MISSIO~1\AppData\Local\Temp\8e537f3f1075ad5ff3f7e8ba8228812c.mpf_cache
2020-04-11 17:35:05,799 : WARNING : ConfigProcessor : Cache file not found: C:\Users\MISSIO~1\AppData\Local\Temp\0db36da4b9aa41dcdeae696ef4ec8d87.mpf_cache
2020-04-11 17:35:05,799 : INFO : ConfigProcessor : Loading config from file E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\gym8_ended\config\gym8_ended.yaml.
2020-04-11 17:35:05,924 : INFO : ConfigProcessor : Loading config: E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\gym8_ended\config\gym8_ended.yaml
2020-04-11 17:35:05,924 : INFO : ConfigProcessor : Config file cache created: C:\Users\MISSIO~1\AppData\Local\Temp\0db36da4b9aa41dcdeae696ef4ec8d87.mpf_cache
2020-04-11 17:35:05,955 : WARNING : ConfigProcessor : Cache file not found: C:\Users\MISSIO~1\AppData\Local\Temp\1a49828cb8454c90fd195e3886327b9d.mpf_cache
2020-04-11 17:35:05,955 : INFO : ConfigProcessor : Loading config from file E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\gym6\config\gym6.yaml.
2020-04-11 17:35:06,143 : INFO : ConfigProcessor : Loading config: E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\gym6\config\gym6.yaml
2020-04-11 17:35:06,143 : INFO : ConfigProcessor : Config file cache created: C:\Users\MISSIO~1\AppData\Local\Temp\1a49828cb8454c90fd195e3886327b9d.mpf_cache
2020-04-11 17:35:06,189 : WARNING : ConfigProcessor : Cache file not found: C:\Users\MISSIO~1\AppData\Local\Temp\b51fa6c1189029eac07bf57001e3c2a0.mpf_cache
2020-04-11 17:35:06,205 : INFO : ConfigProcessor : Loading config from file E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\evolve_pikachu\config\evolve_pikachu.yaml.
2020-04-11 17:35:06,518 : INFO : ConfigProcessor : Loading config: E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\evolve_pikachu\config\evolve_pikachu.yaml
2020-04-11 17:35:06,518 : INFO : ConfigProcessor : Config file cache created: C:\Users\MISSIO~1\AppData\Local\Temp\b51fa6c1189029eac07bf57001e3c2a0.mpf_cache
2020-04-11 17:35:06,564 : WARNING : ConfigProcessor : Cache file not found: C:\Users\MISSIO~1\AppData\Local\Temp\f9a13da868835514a4e555019188716d.mpf_cache
2020-04-11 17:35:06,564 : INFO : ConfigProcessor : Loading config from file E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\gym5_ended\config\gym5_ended.yaml.
2020-04-11 17:35:06,689 : INFO : ConfigProcessor : Loading config: E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\gym5_ended\config\gym5_ended.yaml
2020-04-11 17:35:06,705 : INFO : ConfigProcessor : Config file cache created: C:\Users\MISSIO~1\AppData\Local\Temp\f9a13da868835514a4e555019188716d.mpf_cache
2020-04-11 17:35:06,736 : WARNING : ConfigProcessor : Cache file not found: C:\Users\MISSIO~1\AppData\Local\Temp\5b0c10fb1ea3d696f3389c798402fbbd.mpf_cache
2020-04-11 17:35:06,736 : INFO : ConfigProcessor : Loading config from file E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\gym6_ended\config\gym6_ended.yaml.
2020-04-11 17:35:06,861 : INFO : ConfigProcessor : Loading config: E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\gym6_ended\config\gym6_ended.yaml
2020-04-11 17:35:06,861 : INFO : ConfigProcessor : Config file cache created: C:\Users\MISSIO~1\AppData\Local\Temp\5b0c10fb1ea3d696f3389c798402fbbd.mpf_cache
2020-04-11 17:35:06,893 : INFO : ConfigProcessor : Loading config from cache: C:\Users\MISSIO~1\AppData\Local\Temp\c721f48534ae43d3f9c4e17312560aaa.mpf_cache
2020-04-11 17:35:06,986 : WARNING : ConfigProcessor : Cache file not found: C:\Users\MISSIO~1\AppData\Local\Temp\a79159ab091a1f38ad921bad9a429f77.mpf_cache
2020-04-11 17:35:06,986 : INFO : ConfigProcessor : Loading config from file c:\users\mission pinball\appdata\local\programs\python\python35\lib\site-packages\mpf\modes\attract\config\attract.yaml.
2020-04-11 17:35:07,049 : INFO : ConfigProcessor : Loading config: c:\users\mission pinball\appdata\local\programs\python\python35\lib\site-packages\mpf\modes\attract\config\attract.yaml
2020-04-11 17:35:07,049 : INFO : ConfigProcessor : Loading config from file E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\attract\config\attract.yaml.
2020-04-11 17:35:07,377 : INFO : ConfigProcessor : Loading config: E:\POKEMON - 4 - ATTRACT AND HIGH SCORE DISPLAYS\modes\attract\config\attract.yaml
2020-04-11 17:35:07,377 : ERROR : asyncio : Exception in callback EventManager._async_handler_done(<QueuedEvent>)(<Task finishe...': '20s'}]",)>) at c:\users\mission pinball\appdata\local\programs\python\python35\lib\site-packages\mpf\core\events.py:96
handle: <Handle EventManager._async_handler_done(<QueuedEvent>)(<Task finishe...': '20s'}]",)>) at c:\users\mission pinball\appdata\local\programs\python\python35\lib\site-packages\mpf\core\events.py:96>
Traceback (most recent call last):
  File "c:\users\mission pinball\appdata\local\programs\python\python35\lib\asyncio\events.py", line 127, in _run
    self._callback(*self._args)
  File "c:\users\mission pinball\appdata\local\programs\python\python35\lib\site-packages\mpf\core\events.py", line 99, in _async_handler_done
    future.result()
  File "c:\users\mission pinball\appdata\local\programs\python\python35\lib\asyncio\futures.py", line 294, in result
    raise self._exception
  File "c:\users\mission pinball\appdata\local\programs\python\python35\lib\asyncio\tasks.py", line 240, in _step
    result = coro.send(None)
  File "c:\users\mission pinball\appdata\local\programs\python\python35\lib\site-packages\mpf\core\mode_controller.py", line 117, in load_modes
    self.machine.modes[mode] = self._load_mode(mode)
  File "c:\users\mission pinball\appdata\local\programs\python\python35\lib\site-packages\mpf\core\mode_controller.py", line 266, in _load_mode
    config = self._load_mode_config(mode_string)
  File "c:\users\mission pinball\appdata\local\programs\python\python35\lib\site-packages\mpf\core\mode_controller.py", line 184, in _load_mode_config
    store_to_cache=self.machine.options['create_config_cache'])
  File "c:\users\mission pinball\appdata\local\programs\python\python35\lib\site-packages\mpf\core\config_processor.py", line 112, in load_config_files_with_cache
    ignore_unknown_sections)
  File "c:\users\mission pinball\appdata\local\programs\python\python35\lib\site-packages\mpf\core\config_processor.py", line 142, in _load_config_file_and_return_loaded_files
    raise ConfigFileError("Config should be a dict: {}".format(config), self.log.name, "ConfigProcessor")
mpf.exceptions.ConfigFileError.ConfigFileError: Config File Error in ConfigProcessor: Config should be a dict: [{'slides': {'show_high_scores': {'widgets': [{'font_size': 60, 'type': 'Text', 'bold': True, 'anchor_x': 'center', 'y': 'top-100', 'text': 'HIGH SCORES', 'anchor_y': 'center', 'x': 'center'}, {'y': 'top-200', 'text': '(machine|score1_label)', 'type': 'Text', 'style': 'attract_mode_high_score_display_label'}, {'y': 'top-200', 'text': '(machine|score1_name)', 'type': 'Text', 'style': 'attract_mode_high_score_display_name'}, {'y': 'top-200', 'text': '(machine|score1_value)', 'type': 'Text', 'style': 'attract_mode_high_score_display_score'}, {'y': 'top-240', 'text': '(machine|score2_label)', 'type': 'Text', 'style': 'attract_mode_high_score_display_label'}, {'y': 'top-240', 'text': '(machine|score2_name)', 'type': 'Text', 'style': 'attract_mode_high_score_display_name'}, {'y': 'top-240', 'text': '(machine|score2_value)', 'type': 'Text', 'style': 'attract_mode_high_score_display_score'}, {'y': 'top-280', 'text': '(machine|score3_label)', 'type': 'Text', 'style': 'attract_mode_high_score_display_label'}, {'y': 'top-280', 'text': '(machine|score3_name)', 'type': 'Text', 'style': 'attract_mode_high_score_display_name'}, {'y': 'top-280', 'text': '(machine|score3_value)', 'type': 'Text', 'style': 'attract_mode_high_score_display_score'}, {'font_size': 60, 'type': 'Text', 'bold': True, 'anchor_x': 'center', 'y': 'top-500', 'text': 'LOOP CHAMPION', 'anchor_y': 'center', 'x': 'center'}, {'y': 'top-600', 'text': '(machine|loops1_label)', 'type': 'Text', 'style': 'attract_mode_high_score_display_label'}, {'y': 'top-600', 'text': '(machine|loops1_name)', 'type': 'Text', 'style': 'attract_mode_high_score_display_name'}, {'y': 'top-600', 'text': '(machine|loops1_value)', 'type': 'Text', 'style': 'attract_mode_high_score_display_score'}]}}, 'duration': '20s'}] Error Code: CFE-ConfigProcessor-ConfigProcessor
2020-04-11 17:35:07,721 : INFO : Machine : Starting the main run loop.
2020-04-11 17:35:07,721 : INFO : Machine : Shutting down...
2020-04-11 17:35:07,721 : INFO : EventManager : Event: ======'shutdown'====== Args={}
2020-04-11 17:35:07,736 : ERROR : FAST : Stop called on serial connection com5
2020-04-11 17:35:07,752 : ERROR : FAST : Stop called on serial connection com4
2020-04-11 17:35:08,033 : ERROR : root : Config File Error in ConfigProcessor: Config should be a dict: [{'slides': {'show_high_scores': {'widgets': [{'font_size': 60, 'type': 'Text', 'bold': True, 'anchor_x': 'center', 'y': 'top-100', 'text': 'HIGH SCORES', 'anchor_y': 'center', 'x': 'center'}, {'y': 'top-200', 'text': '(machine|score1_label)', 'type': 'Text', 'style': 'attract_mode_high_score_display_label'}, {'y': 'top-200', 'text': '(machine|score1_name)', 'type': 'Text', 'style': 'attract_mode_high_score_display_name'}, {'y': 'top-200', 'text': '(machine|score1_value)', 'type': 'Text', 'style': 'attract_mode_high_score_display_score'}, {'y': 'top-240', 'text': '(machine|score2_label)', 'type': 'Text', 'style': 'attract_mode_high_score_display_label'}, {'y': 'top-240', 'text': '(machine|score2_name)', 'type': 'Text', 'style': 'attract_mode_high_score_display_name'}, {'y': 'top-240', 'text': '(machine|score2_value)', 'type': 'Text', 'style': 'attract_mode_high_score_display_score'}, {'y': 'top-280', 'text': '(machine|score3_label)', 'type': 'Text', 'style': 'attract_mode_high_score_display_label'}, {'y': 'top-280', 'text': '(machine|score3_name)', 'type': 'Text', 'style': 'attract_mode_high_score_display_name'}, {'y': 'top-280', 'text': '(machine|score3_value)', 'type': 'Text', 'style': 'attract_mode_high_score_display_score'}, {'font_size': 60, 'type': 'Text', 'bold': True, 'anchor_x': 'center', 'y': 'top-500', 'text': 'LOOP CHAMPION', 'anchor_y': 'center', 'x': 'center'}, {'y': 'top-600', 'text': '(machine|loops1_label)', 'type': 'Text', 'style': 'attract_mode_high_score_display_label'}, {'y': 'top-600', 'text': '(machine|loops1_name)', 'type': 'Text', 'style': 'attract_mode_high_score_display_name'}, {'y': 'top-600', 'text': '(machine|loops1_value)', 'type': 'Text', 'style': 'attract_mode_high_score_display_score'}]}}, 'duration': '20s'}] Error Code: CFE-ConfigProcessor-ConfigProcessor
Traceback (most recent call last):
  File "c:\users\mission pinball\appdata\local\programs\python\python35\lib\site-packages\mpf\commands\game.py", line 220, in __init__
    MachineController(mpf_path, machine_path, vars(self.args)).run()
  File "c:\users\mission pinball\appdata\local\programs\python\python35\lib\site-packages\mpf\core\machine.py", line 709, in run
    self._run_loop()
  File "c:\users\mission pinball\appdata\local\programs\python\python35\lib\site-packages\mpf\core\machine.py", line 769, in _run_loop
    raise self._exception['exception']
  File "c:\users\mission pinball\appdata\local\programs\python\python35\lib\asyncio\events.py", line 127, in _run
    self._callback(*self._args)
  File "c:\users\mission pinball\appdata\local\programs\python\python35\lib\site-packages\mpf\core\events.py", line 99, in _async_handler_done
    future.result()
  File "c:\users\mission pinball\appdata\local\programs\python\python35\lib\asyncio\futures.py", line 294, in result
    raise self._exception
  File "c:\users\mission pinball\appdata\local\programs\python\python35\lib\asyncio\tasks.py", line 240, in _step
    result = coro.send(None)
  File "c:\users\mission pinball\appdata\local\programs\python\python35\lib\site-packages\mpf\core\mode_controller.py", line 117, in load_modes
    self.machine.modes[mode] = self._load_mode(mode)
  File "c:\users\mission pinball\appdata\local\programs\python\python35\lib\site-packages\mpf\core\mode_controller.py", line 266, in _load_mode
    config = self._load_mode_config(mode_string)
  File "c:\users\mission pinball\appdata\local\programs\python\python35\lib\site-packages\mpf\core\mode_controller.py", line 184, in _load_mode_config
    store_to_cache=self.machine.options['create_config_cache'])
  File "c:\users\mission pinball\appdata\local\programs\python\python35\lib\site-packages\mpf\core\config_processor.py", line 112, in load_config_files_with_cache
    ignore_unknown_sections)
  File "c:\users\mission pinball\appdata\local\programs\python\python35\lib\site-packages\mpf\core\config_processor.py", line 142, in _load_config_file_and_return_loaded_files
    raise ConfigFileError("Config should be a dict: {}".format(config), self.log.name, "ConfigProcessor")
mpf.exceptions.ConfigFileError.ConfigFileError: Config File Error in ConfigProcessor: Config should be a dict: [{'slides': {'show_high_scores': {'widgets': [{'font_size': 60, 'type': 'Text', 'bold': True, 'anchor_x': 'center', 'y': 'top-100', 'text': 'HIGH SCORES', 'anchor_y': 'center', 'x': 'center'}, {'y': 'top-200', 'text': '(machine|score1_label)', 'type': 'Text', 'style': 'attract_mode_high_score_display_label'}, {'y': 'top-200', 'text': '(machine|score1_name)', 'type': 'Text', 'style': 'attract_mode_high_score_display_name'}, {'y': 'top-200', 'text': '(machine|score1_value)', 'type': 'Text', 'style': 'attract_mode_high_score_display_score'}, {'y': 'top-240', 'text': '(machine|score2_label)', 'type': 'Text', 'style': 'attract_mode_high_score_display_label'}, {'y': 'top-240', 'text': '(machine|score2_name)', 'type': 'Text', 'style': 'attract_mode_high_score_display_name'}, {'y': 'top-240', 'text': '(machine|score2_value)', 'type': 'Text', 'style': 'attract_mode_high_score_display_score'}, {'y': 'top-280', 'text': '(machine|score3_label)', 'type': 'Text', 'style': 'attract_mode_high_score_display_label'}, {'y': 'top-280', 'text': '(machine|score3_name)', 'type': 'Text', 'style': 'attract_mode_high_score_display_name'}, {'y': 'top-280', 'text': '(machine|score3_value)', 'type': 'Text', 'style': 'attract_mode_high_score_display_score'}, {'font_size': 60, 'type': 'Text', 'bold': True, 'anchor_x': 'center', 'y': 'top-500', 'text': 'LOOP CHAMPION', 'anchor_y': 'center', 'x': 'center'}, {'y': 'top-600', 'text': '(machine|loops1_label)', 'type': 'Text', 'style': 'attract_mode_high_score_display_label'}, {'y': 'top-600', 'text': '(machine|loops1_name)', 'type': 'Text', 'style': 'attract_mode_high_score_display_name'}, {'y': 'top-600', 'text': '(machine|loops1_value)', 'type': 'Text', 'style': 'attract_mode_high_score_display_score'}]}}, 'duration': '20s'}] Error Code: CFE-ConfigProcessor-ConfigProcessor
